src/CNN/networks/Training.py

```diff
@@ -210,25 +211,19 @@ def fit(self, epochs, train_path, result_datapath, kwargs=None, n_datapoints=376
         train_rmse = []
         test_rmse = []
         if kwargs is None:
-            kwargs = {'num_workers': 4}
-
-        indices = np.arange(n_datapoints)
-        train_size = int(prct_train * n_datapoints)
-        test_size = n_datapoints - train_size
-        train, test = torch.utils.data.random_split(indices, [train_size, test_size])
-        train_set = OwnDataset(train, train_path, rotations=augmentation)
+            kwargs = {'num_workers': 8}
 
-        if test_path is None:
-            test_set = OwnDataset(test, train_path, rotations=False)
-        else:
-            test_set = OwnDataset(np.arange(n_test), test_path, rotations=False)
+#         indices = np.arange(n_datapoints)
+#         train_size = int(prct_train * n_datapoints)
+#         test_size = n_datapoints - train_size
+#         train, test = torch.utils.data.random_split(indices, [train_size, test_size])
+        # Use the entire refined set as train set and core set as test/validation set
```

Comment:
The Test-set and Validation-set should never be the same.

Reply:
I am afraid the Kdeep paper used the test set as validation set!!